resources/integration_test/tests/queries.yaml

```diff
@@ -67,6 +67,15 @@ tests:
     expected_scale: [ 0, 0, 0, 3, 0 ]
     expected_is_nullable: [ 1, 1, 1, 1, 1 ]
 
+  - description: select_star_unordered_using_simple_types_only_mode
+    test_definition: SELECT * FROM class
+    is_simple_type: true
+    db: integration_test
+    expected_display_size: [ 4, 1, 0, 16, 4 ]
+    expected_length: [ 4, 1, 0, 16, 4 ]
+    expected_sql_type: [ 4, -7, -9, 93, 4 ]
+    expected_precision: [ 10, 1, 0, 23, 10 ]
+
```

Comment:
The `simple_types_only` version of this test expects the same results as the regular version.

Reply:
Yes, I saw the change.
It's not how I would have done it. Because, even though you know today that the types info did not change, nothing will show this in the long run.
Separate metadata tests would have been better. What we would really need is a complete coverage of all data types in 2 tests. 
However, the tests you added for complex types has a simple type column and we see it does not change.
And with the tests you added, we achieve the coverage, only not with targeted tests, which can make things harder to grasp when it fails.